# Translations template for keystone.
# Copyright (C) 2015 OpenStack Foundation
# This file is distributed under the same license as the keystone project.
#
# Translators:
# OpenStack Infra <zanata@openstack.org>, 2015. #zanata
msgid ""
msgstr ""
"Project-Id-Version: keystone 8.0.1.dev11\n"
"Report-Msgid-Bugs-To: https://bugs.launchpad.net/keystone\n"
"POT-Creation-Date: 2015-11-05 06:13+0000\n"
"PO-Revision-Date: 2015-08-04 01:50+0000\n"
"Last-Translator: İşbaran Akçayır <isbaran@gmail.com>\n"
"Language-Team: Turkish (Turkey)\n"
"Language: tr-TR\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"Generated-By: Babel 2.0\n"
"X-Generator: Zanata 3.7.1\n"

msgid "Cannot retrieve Authorization headers"
msgstr "Yetkilendirme başlıkları alınamıyor"

#, python-format
msgid ""
"Circular reference or a repeated entry found in projects hierarchy - "
"%(project_id)s."
msgstr ""
"Proje sıra düzeninde çember başvuru ya da tekrar eden girdi bulundu - "
"%(project_id)s."

#, python-format
msgid ""
"Circular reference or a repeated entry found in region tree - %(region_id)s."
msgstr ""
"Bölge ağacında çember başvuru ya da tekrar eden girdi bulundu - "
"%(region_id)s."

#, python-format
msgid ""
"Circular reference or a repeated entry found projects hierarchy - "
"%(project_id)s."
msgstr ""
"Proje sıra düzeninde çember başvuru ya da tekrar eden girdi bulundu - "
"%(project_id)s."

#, python-format
msgid "Command %(to_exec)s exited with %(retcode)s- %(output)s"
msgstr "%(to_exec)s komutu %(retcode)s ile çıktı- %(output)s"

#, python-format
msgid "Could not bind to %(host)s:%(port)s"
msgstr "%(host)s:%(port)s adresine bağlanılamadı"

#, python-format
msgid ""
"Either [fernet_tokens] key_repository does not exist or Keystone does not "
"have sufficient permission to access it: %s"
msgstr ""
"[fernet_tokents] key_repository mevcut değil ya da Keystone erişmek için "
"yeterli izine sahip değil: %s"

msgid ""
"Error setting up the debug environment. Verify that the option --debug-url "
"has the format <host>:<port> and that a debugger processes is listening on "
"that port."
msgstr ""
"Hata ayıklama ortamının ayarlanmasında hata. --debug-url seçeneğinin "
"<istemci>:<bağlantı noktası> biçimine sahip olduğunu ve bu bağlantı "
"noktasında hata ayıklama sürecinin dinlediğini doğrulayın."

msgid "Failed to construct notifier"
msgstr "Bildirici inşa etme başarısız"

msgid ""
"Failed to create [fernet_tokens] key_repository: either it already exists or "
"you don't have sufficient permissions to create it"
msgstr ""
"[fernet_tokens] key_repository oluşturulamıyor: ya zaten mevcut ya da "
"oluşturmak için yeterli izniniz yok"

#, python-format
msgid "Failed to remove file %(file_path)r: %(error)s"
msgstr "%(file_path)r dosyası silinemedi: %(error)s"

#, python-format
msgid "Failed to send %(action)s %(event_type)s notification"
msgstr "%(action)s %(event_type)s bildirimi gönderilemedi"

#, python-format
msgid "Failed to send %(res_id)s %(event_type)s notification"
msgstr "%(res_id)s %(event_type)s bildirimi gönderilemedi"

msgid "Failed to validate token"
msgstr "Jeton doğrulama başarısız"

#, python-format
msgid "Malformed endpoint %(url)s - unknown key %(keyerror)s"
msgstr "Kusurlu bitiş noktası %(url)s - bilinmeyen anahtar %(keyerror)s"

#, python-format
msgid ""
"Malformed endpoint %s - incomplete format (are you missing a type notifier ?)"
msgstr ""
"Kusurlu bitiş noktası %s - tamamlanmamış biçim (bir tür bildiriciniz eksik "
"olabilir mi ?)"

#, python-format
msgid ""
"Malformed endpoint '%(url)s'. The following type error occurred during "
"string substitution: %(typeerror)s"
msgstr ""
"Kusurlu bitiş noktası '%(url)s'. Karakter dizisi yer değiştirme sırasında şu "
"tür hatası oluştu: %(typeerror)s"

#, python-format
msgid "Malformed endpoint - %(url)r is not a string"
msgstr "Kusurlu bitiş noktası - %(url)r bir karakter dizisi değil"

#, python-format
msgid ""
"Reinitializing revocation list due to error in loading revocation list from "
"backend.  Expected `list` type got `%(type)s`. Old revocation list data: "
"%(list)r"
msgstr ""
"Arka uçtan feshetme listesi yüklemedeki hata sebebiyle fesih listesi yeniden "
"ilklendiriliyor.  `list` beklendi `%(type)s` alındı. Eski fesih listesi "
"verisi: %(list)r"

msgid "Server error"
msgstr "Sunucu hatası"

#, python-format
msgid ""
"Unable to build cache config-key. Expected format \"<argname>:<value>\". "
"Skipping unknown format: %s"
msgstr ""
"Zula yapılandırma anahtarı inşa edilemiyor. Beklenen biçim \"<değişken ismi>:"
"<değer>\". Bilinmeyen biçim atlanıyor: %s"

#, python-format
msgid "Unable to convert Keystone user or group ID. Error: %s"
msgstr "Keystone kullanıcı veya grup kimliği dönüştürülemiyor. Hata: %s"

msgid "Unable to sign token"
msgstr "Jeton imzalanamıyor"

#, python-format
msgid "Unexpected error or malformed token determining token expiry: %s"
msgstr "Jeton sona erme belirlemede beklenmeyen hata veya kusurlu jeton: %s"

#, python-format
msgid ""
"Unexpected results in response for domain config - %(count)s responses, "
"first option is %(option)s, expected option %(expected)s"
msgstr ""
"Alan yapılandırması yanıtında beklenmedik sonuçlar - %(count)s yanıt, ilk "
"seçenek %(option)s, beklenen seçenek %(expected)s"
